sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTestTest.java

```diff
@@ -162,6 +162,349 @@ void testDown() throws TimeoutException {
+        messageStorageUtil.addInboundSecureHttpMessage(storage, second);
+
+        final Envelope third = buildDescriptionModificationReport(
+                SEQUENCE_ID,
+                BigInteger.valueOf(3),
+                buildDescriptionModificationReportPart(
+                        DescriptionModificationType.CRT,
+                        parentDescriptor,
+                        mdibBuilder.buildSystemContext(SYSTEM_CONTEXT_HANDLE),
+                        mdibBuilder.buildSco(SCO_HANDLE)));
+        messageStorageUtil.addInboundSecureHttpMessage(storage, third);
+
+        assertThrows(AssertionError.class, testClass::testRequirementR00550);
+    }
+
+    /**
+     * Checks whether a DescriptionModificationReport with a ReportPart that has ParentDescriptor=null
+     * passes the test when the ReportPart contains only MdsDescriptors.
+     *
+     * @throws Exception on any exception
+     */
+    @Test
+    public void testRequirementR00550GoodParentDescriptorNullButOnlyMdsDescriptors() throws Exception {
+        final Envelope initial = buildMdib(SEQUENCE_ID, BigInteger.ZERO);
+        messageStorageUtil.addInboundSecureHttpMessage(storage, initial);
+
+        final Pair<MdsDescriptor, MdsState> mdsDescriptor = mdibBuilder.buildMds(MdibBuilder.DEFAULT_MDS_HANDLE);
+
+        final String parentDescriptor = "parentDescriptor";
+        final Envelope first = buildDescriptionModificationReport(
+                SEQUENCE_ID,
+                BigInteger.ONE,
+                buildDescriptionModificationReportPart(DescriptionModificationType.CRT, (String) null, mdsDescriptor));
+        messageStorageUtil.addInboundSecureHttpMessage(storage, first);
+
+        final Envelope second = buildDescriptionModificationReport(
+                SEQUENCE_ID,
+                BigInteger.TWO,
+                buildDescriptionModificationReportPart(
+                        DescriptionModificationType.CRT,
+                        parentDescriptor,
+                        mdibBuilder.buildChannel("second channel"),
+                        mdibBuilder.buildVmd("second vmd")));
+        messageStorageUtil.addInboundSecureHttpMessage(storage, second);
+
+        final Envelope third = buildDescriptionModificationReport(
+                SEQUENCE_ID,
+                BigInteger.valueOf(3),
+                buildDescriptionModificationReportPart(
+                        DescriptionModificationType.CRT,
+                        parentDescriptor,
+                        mdibBuilder.buildSystemContext(SYSTEM_CONTEXT_HANDLE),
+                        mdibBuilder.buildSco(SCO_HANDLE)));
+        messageStorageUtil.addInboundSecureHttpMessage(storage, third);
+
+        testClass.testRequirementR00550();
+    }
+
+    /**
+     * Checks whether a sequence of DescriptionModificationReports,
+     * in which one ReportPart
+     * has @ParentDescriptor=null, but contains no Descriptors
+     * passes the test.
```

Comment:
The report part with parent descriptor null and without descriptors is not failing the test instead of passing it.

Reply:
Rephrased.